cypress/e2e/shuffle.cy.ts

```diff
@@ -79,6 +79,39 @@ describe('Shuffle', () => {
         });
     });
 
+    it('displays item with children shuffled when redirected', () => {
+      const root = FOLDER_WITH_FIVE_ORDERED_SUBFOLDER_ITEMS.items[0];
+      cy.visit(
+        buildContentPagePath({
+          rootId: root.id,
+          itemId: '',
+          searchParams: 'shuffle=true',
+        }),
```

Comment:
There is also a function that sets only the `rootId`: `buildMainPath`, you would just need to add the shuffle param.

Reply:
@spaenleh is this fixed or should I fix it in the new PR #690?

Reply:
This is ok, it was more of a suggestion.